README.md

```diff
@@ -43,7 +43,12 @@ This will make certbot install any additional packages it needs and create its e
 By installing Certbot via packages it automatically creates a cron schedule and a systemd timer to renew certificates (at least on Ubuntu). 
 We must **disable this schedule** because after the renew we must deploy it in Zimbra. Also certbot's timers will attempt to update the cert twice a day,
 this means a Zimbra restart may happen during work hours.
-So open `/etc/cron.d/certbot` with your favourite editor and **comment the last line**. To disable systemd timers run:
+So open `/etc/cron.d/certbot` with your favourite editor and **comment the last line** or use the below sed command to do so as root. 
+```
+sed -i '/certbot -q renew/s/^/#/g' /etc/cron.d/certbot
```

Comment:
I was testing this and seems like the file usually has something like the below in it from the server's ive seen.
```
0 */12 * * * root test -x /usr/bin/certbot -a \! -d /run/systemd/system && perl -e 'sleep int(rand(43200))' && certbot -q renew
```
Command works to do this.
```
#0 */12 * * * root test -x /usr/bin/certbot -a \! -d /run/systemd/system && perl -e 'sleep int(rand(43200))' && certbot -q renew
```